fluss-server/src/test/java/com/alibaba/fluss/server/kv/prewrite/KvPreWriteBufferTest.java

```diff
@@ -165,7 +165,7 @@ void testTruncate() {
         bufferPut(buffer, "key1", "value1-1", elementCount++);
         assertThat(getValue(buffer, "key1")).isEqualTo("value1-1");
         assertThat(buffer.getMaxLSN()).isEqualTo(elementCount - 1);
-        buffer.truncateTo(5, "test");
+        buffer.truncateTo(5, KvPreWriteBuffer.KvPreWriteBufferTruncateReason.ERROR);
```

Comment:
nit: use on-demand or static import for the enum.

Reply:
Done!